---
title: "Changde Cheng Lab"
listing:
  contents: preprints
  sort: "date asc"
  type: default
  categories: false
  sort-ui: false
  filter-ui: false
# listing:
#   contents: posts
#   sort: "date desc"
#   type: default
#   fields: [date, title, author, reading-time]
# #  fields: [date, title]
#   categories: true
#   sort-ui: false
#   filter-ui: false
page-layout: full
title-block-banner: false
about:
  id: about
  template: jolla
  image: logo.png
embed-resources: true
---

::: {#about}
:::

<!-- Dr. Changde Cheng uses computational and analytical methods to investigate cancer ecology and evolution's impact on therapeutic resistance and side effects. The primary objective is to enhance cancer patient outcomes. -->

## Computational Biology and Cancer Genomics

We investigate the impact of (epi-)mutations and cell-cell communication in cancer genomics using computational and analytical methods.  We develop innovative analytical methods and machine-learning tools to advance our understanding of cancer biology in patients.  We take high-throughput approaches, including single-cell RNA-seq, spatial, multiomics, and perturbation sequencing, to study cancer's origin, progression, and therapeutic resistance.  Our research focuses on identifying the cancer cell subpopulations responsible for relapse and understanding how microenvironments influence their response to treatment by decoding communication networks among cells. 

Our ultimate goal is to translate our research findings into clinical settings to improve cancer treatment precision, minimize disease recurrence risk, and enhance patient outcomes.

## List of working papers

---
<!-- **Methods development and validation** - Our laboratory aims to create innovative computational methods and machine learning tools that can utilize high-throughput, high-dimensional genomics and heterogeneous biological data to gain a comprehensive and in-depth understanding of the biology of patient tumors. This understanding will help us develop effective treatment and therapy options, which can be translated into clinical practice. -->

<!-- **Representative publications** -->

<!-- **Single-cell, spatial, and multi-omics for cancer research**.  Our research leverages high-throughput techniques such as single-cell RNA-seq, spatial, multiomics, and  perturbation sequencing to investigate cancer initiation, progression, and therapeutic resistance. We focus on identifying specific tumor cell subpopulations responsible for relapse and understanding how microenvironments influence their response to chemotherapy. We've developed high-performance machine-learning tools for single-cell analysis, enabling accurate subpopulation estimation and clustering. This allows us to define disease-causing cell states, predict their behavior under different conditions, and understand cell interactions. Our goal is to translate these insights into clinical practice for better identifying patients at risk of disease recurrence, thus improving the precision treatment and outcome of cancer. -->

<!-- 1.	Cheng C, Easton J, Rosencrance C, Li Y, Ju B, Williams J, Mulder HL, Pang Y, Chen W, Chen X. Latent cellular analysis robustly reveals subtle diversity in large-scale single-cell RNA-seq data. *Nucleic Acids Res*. 2019 Dec 16;47(22):e143.  -->
<!-- 2.	Tamura T,$^{\ast}$ Cheng C,$^{\ast}$ Chen W, Merriam LT, Athar H, Kim YH, Manandhar R, Amir Sheikh MD, Pinilla-Vera M, Varon J, Hou PC, Lawler PR, Oldham WM, Seethala RR, Tesfaigzi Y, Weissman AJ, Baron RM, Ichinose F, Berg KM, Bohula EA, Morrow DA, Chen X, Kim EY. Single-cell transcriptomics reveal a hyperacute cytokine and immune checkpoint axis after cardiac arrest in patients with poor neurological outcome. *Med*. 2023 Jul 14;4(7):432-456.e6. ($^{\ast}$Equal contribution) -->
<!-- 3.	Fang J,$^{\ast}$ Singh S,$^{\ast}$ Cheng C,$^{\ast}$ Natarajan S,$^{\ast}$ Sheppard H,$^{\ast}$ Abu-Zaid A, Durbin AD, Lee HW, Wu Q, Steele J, Connelly JP, Jin H, Chen W, Fan Y, Pruett-Miller SM, Rehg JE, Koo SC, Santiago T, Emmons J, Cairo S, Wang R, Glazer ES, Murphy AJ, Chen T, Davidoff AM, Armengol C, Easton J, Chen X, Yang J. Genome-wide mapping of cancer dependency genes and genetic modifiers of chemotherapy in high-risk hepatoblastoma. *Nat Commun*. 2023 Jul 6;14(1):4003. ($^{\ast}$Equal contribution) -->
<!-- 4.	Yang K, Blanco DB, Chen X, Dash P, Neale G, Rosencrance C, Easton J, Chen W, Cheng C, Dhungana Y, Kc A, Awad W, Guo XJ, Thomas PG, Chi H. Metabolic signaling directs the reciprocal lineage decisions of αβ and γδ T cells. *Sci Immunol*. 2018 Jul 6;3(25).  -->


<!-- **Statistical and populational genetics of aging and complex quantitative traits**.   Why do we age? What is the evolutionary origin of cancer? Why do females differ from males? What are the evolutionary forces shaping the differences between the sexes? I have used statistical and population genetics modeling and genomics data analysis to tackle these fundamental questions. We study the relation between molecular evolution, genetic variation patterns, and age-dependent gene expression, and our findings suggest that selection against deleterious mutations gets less efficient with age. We developed a population genetics model to show a high risk of loss of function when the strength of selection is weak. This may explain why late-expressed genes appear to have short lifespans and are more likely to contribute to cancer. For the evolution of sexual dimorphism, we have developed a theory suggesting that the canonical scenario for the evolution of sexual dimorphism by antagonistic selection may be less widely applicable than believed. We used a geometrical approach to show how the evolution of dimorphism responds to selection under a strong genetic constraint.  -->

<!-- 1.	Cheng C, Kirkpatrick M. Molecular evolution and the decline of purifying selection with age. *Nat Commun*. 2021 May 11;12(1):2657.  -->
<!-- 2.	Cheng C, Houle D. Predicting Multivariate Responses of Sexual Dimorphism to Direct and Indirect Selection. *Am Nat*. 2020 Oct;196(4):391-405. -->
<!-- 3.	Houle D, Cheng C. Predicting the Evolution of Sexual Dimorphism in Gene Expression. *Mol Biol Evol*. 2021 May 4;38(5):1847-1859.  -->
<!-- 4.	Cheng C, Kirkpatrick M. Inversions are bigger on the X chromosome. *Mol Ecol*. 2019 Mar;28(6):1238-1245.  -->
<!-- 5.	Cheng C, Kirkpatrick M. Sex-Specific Selection and Sex-Biased Gene Expression in Humans and Flies. *PLoS Genet*. 2016 Sep;12(9):e1006170.  -->

<!-- Leave it out for lab page -->

<!-- **Systems and evolutionary genomics of sex chromosomes and chromosomal rearrangements**.  -->
<!-- Early in my career, I have played a leading role in discovering conserved male determination genes through evolutionary genomics analysis of malaria mosquitoes' first fully assembled Y chromosome.  Our work is considered the most significant breakthrough in decades and has received wide media coverage in different languages.  I was among the first to study genomic structural variation and its role in evolution and ecology using whole-genome sequencing and systems genomics in natural populations.  I was one of the first to propose that "speciation islands" could be divergent genomic islands with no role in speciation, as supported by our population genomics analysis.   -->

<!-- 1.	Cheng C, Tan JC, Hahn MW, Besansky NJ. Systems genetic analysis of inversion polymorphisms in the malaria mosquito Anopheles gambiae. Proc *Natl Acad Sci U S A*. 2018 Jul 24;115(30):E7005-E7014.  -->
<!-- 2.	Hall AB,$^{\ast}$ Papathanos PA,$^{\ast}$ Sharma A,$^{\ast}$ Cheng C,$^{\ast}$ Akbari OS, Assour L, Bergman NH, Cagnetti A, Crisanti A, Dottorini T, Fiorentini E, Galizi R, Hnath J, Jiang X, Koren S, Nolan T, Radune D, Sharakhova MV, Steele A, Timoshevskiy VA, Windbichler N, Zhang S, Hahn MW, Phillippy AM, Emrich SJ, Sharakhov IV, Tu ZJ, Besansky NJ. Radical remodeling of the Y chromosome in a recent radiation of malaria mosquitoes. *Proc Natl Acad Sci U S A*. 2016 Apr 12;113(15):E2114-23.  ($^{\ast}$Equal contribution) -->
<!-- 3.	Cheng C, White BJ, Kamdem C, Mockaitis K, Costantini C, Hahn MW, Besansky NJ. Ecological genomics of Anopheles gambiae along a latitudinal cline: a population-resequencing approach. *Genetics*. 2012 Apr;190(4):1417-32.  -->
<!-- 4.	White BJ, Cheng C, Simard F, Costantini C, Besansky NJ. Genetic association of physically unlinked islands of genomic divergence in incipient species of Anopheles gambiae. *Mol Ecol*. 2010 Mar;19(5):925-39.   -->
<!-- 5.  White BJ, Cheng C, Sangaré D, Lobo NF, Collins FH, Besansky NJ. The population genomics of trans-specific inversion polymorphisms in Anopheles gambiae. *Genetics*. 2009 Sep;183(1):275-88.  -->


<!-- Before joining UAB, I served as a lead computational research scientist in the computational biology department at the [St. Jude Children's Research Hospital](https://www.stjude.org/). I did my postdoc with [Mark Kirkpatrick](https://en.wikipedia.org/wiki/Mark_Kirkpatrick) at the University of Texas, Austin.  I studied genetics and genomics with [Nora Besansky](https://en.wikipedia.org/wiki/Nora_J._Besansky) at Notre Dame, and ecology and evolutionary biology with [Chuck Taylor](https://dailybruin.com/2013/11/06/ucla-professor-researches-birdsongs-in-mexico-australia) at UCLA. -->
