---
title: "Changde Cheng"
page-layout: article
title-block-banner: false
# about:
#   id: about
#   template: jolla
#   image: logo.png
embed-resources: true
---


<!-- ## Changde Cheng -->

::: {style="font-size: 80%; color:green;"}
Stem Cell Biology Program,  Department of Medicine\
Institute for Cancer Outcomes and Survivorship\
Department of Biomedical Informatics and Data Science\
Center for Clinical and Translational Science\
O'Neal Comprehensive Cancer Center\
Heersink School of Medicine\
University of Alabama at Birmingham
:::

---

## Research Overview

Our investigations use high-dimensional genomic approaches to elucidate the molecular mechanisms underlying cancer progression and therapeutic resistance. Through integrated analysis of single-cell transcriptomics, spatial genomics, and multi-modal omics data, we systematically characterize cellular subpopulations and microenvironmental factors that contribute to disease origin, recurrence and resistance.

* High-Throughput Genomic Analysis
  * Single-cell RNA sequencing for cellular heterogeneity profiling
  * Spatial transcriptomics for microenvironmental context
  * Multi-modal omics integration for comprehensive molecular characterization
  * Perturbation-based functional genomics

* Computational Biology Applications
  * Development of robust statistical frameworks
  * Algorithm optimization for cellular state inference
<!--   * Network-based approaches for intercellular communication -->
<!--   * Integration of heterogeneous data modalities -->

* Theoretical Foundations
  * Population genetics
  <!-- of clonal evolution -->
  * Quantitative genetics and phenotypic variation
  * Systematic analysis of evolutionary forces



## Research Evolution

My research focus evolves over time, starting with extensive work in ecological genomics and infectious disease vectors (2009-2016), transitioning through evolutionary biology and bioinformatics (2016-2022), and most recently focusing on clinical applications in cardiac arrest and cancer biology (2023-2024).



### Immunological Mechanisms in Cardiac Arrest  (2023-2024)

Papers [#25](https://www.ncbi.nlm.nih.gov/pubmed/37257452/), [#28](https://www.sciencedirect.com/science/article/pii/S266616672400039X),  [#31](https://www.science.org/doi/10.1126/scitranslmed.adq5796) \
Focus on immune response and inflammation after cardiac arrest \

* Research Themes
  * single-cell transcriptomics, 
  * immune checkpoint regulation, 
  * inflammation markers 

Notable paper:  [#31](https://www.science.org/doi/10.1126/scitranslmed.adq5796), the latest Science Translational Medicine publication (2024) on T cells.



### Cancer Biology and Therapeutic Response  (2023-2024)

Submissions [№ 2](working.qmd#sec-w2), [№ 3](working.qmd#sec-w3), [№ 4](working.qmd#sec-w4), [№ 5](working.qmd#sec-w5), [№ 6](working.qmd#sec-w6), and
papers [#23](https://www.ncbi.nlm.nih.gov/pubmed/36037042/),  [#24](https://www.ncbi.nlm.nih.gov/pubmed/37414763/), [#27](https://www.nature.com/articles/s41467-023-43730-0), [#29](https://doi.org/10.1200/JCO-24-01487),   [#30](https://doi.org/10.1182/blood.2023023644)

* Research Themes:
  * Wilms tumor genetics and epigenetics
  * Hepatoblastoma genomics and drug response
  * Pancreatic cancer tumorigeneiss and chemotherapy
  * Leukemia treatment (menin inhibitor response)
  * Therapy-related myeloid neoplasms
  * Rhabdomyosarcoma biology




### Computational Biology  and Single-Cell Analysis (2019-2023)

Papers  [#19](https://www.ncbi.nlm.nih.gov/pubmed/31566233/), [#26](https://pubmed.ncbi.nlm.nih.gov/37566049/)

* Focus on computational methods for:
  * Single-cell RNA sequencing analysis
  * Cell annotation and integration
  * Cell-cell communication analysis





### Evolutionary Biology and Genetics (2016-2022)

Submission [№ 1](working.qmd#sec-w1), 
and papers   [#11](https://www.ncbi.nlm.nih.gov/pubmed/27658217/), [#16](https://www.ncbi.nlm.nih.gov/pubmed/29987007/), [#17](https://www.ncbi.nlm.nih.gov/pubmed/30059177/), [#20](https://www.ncbi.nlm.nih.gov/pubmed/32970462/), [#21](https://www.ncbi.nlm.nih.gov/pubmed/33512493/),  [#22](https://www.ncbi.nlm.nih.gov/pubmed/33976227/)

* Research Themes:
  * Molecular evolution and aging
  * Sexual dimorphism
  * Chromosomal inversions, local adaptation, and speciation
  * Sex-specific selection





### Vector Genomics and Infectious Disease Transmission (2009-2016)

Papers [#1](https://www.ncbi.nlm.nih.gov/pubmed/19581444/),  [#2](https://www.ncbi.nlm.nih.gov/pubmed/20149091/), [#3](https://www.ncbi.nlm.nih.gov/pubmed/21173248/), [#4](https://www.ncbi.nlm.nih.gov/pubmed/21535279/), [#5](https://www.ncbi.nlm.nih.gov/pubmed/22209907/), [#6](https://www.ncbi.nlm.nih.gov/pubmed/24721548/), [#7](https://www.ncbi.nlm.nih.gov/pubmed/24673723/), [#8](https://www.ncbi.nlm.nih.gov/pubmed/25920668/), [#9](https://www.ncbi.nlm.nih.gov/pubmed/27035980/), [#10](https://www.ncbi.nlm.nih.gov/pubmed/26945667/), [#12](https://www.ncbi.nlm.nih.gov/pubmed/28945882/)

* Research Themes:
  * *Anopheles gambiae* genetics
  * Malaria vector studies
  * Y chromosome evolution
  * Chromosomal rearrangements
  * Local adaptation and speciation
  * Population genomics




### Plant Biology (2019)

Paper [#18](https://www.ncbi.nlm.nih.gov/pubmed/30847928/)

Focus on gene expression patterns in C4 grass related to photoperiod response

  * Diurnal transcriptional profiling


