---
title: "Latent cellular analysis robustly reveals subtle diversity in large-scale single-cell RNA-seq data"
date: 2019-09-30
#authors: ["C. Cheng", "J. Easton", "C. Rosencrance", "Y. Li", "B. Ju", "J. Williams", "H. L. Mulder", "Y. Pang", "W. Chen", "X. Chen"]
publication: "*Nucleic Acids Res*"
categories: ["Nucleic Acids Research"]
url_pdf: "https://www.ncbi.nlm.nih.gov/pubmed/31566233"
doi: "10.1093/nar/gkz826"
image: featured.jpg
---

### Abstract

Single-cell RNA sequencing (scRNA-seq) is a powerful tool for characterizing the cell-to-cell variation and cellular dynamics in populations which appear homogeneous otherwise in basic and translational biological research. However, significant challenges arise in the analysis of scRNA-seq data, including the low signal-to-noise ratio with high data sparsity, potential batch effects, scalability problems when hundreds of thousands of cells are to be analyzed among others. The inherent complexities of scRNA-seq data and dynamic nature of cellular processes lead to suboptimal performance of many currently available algorithms, even for basic tasks such as identifying biologically meaningful heterogeneous subpopulations. 

In this study, we developed the Latent Cellular Analysis (LCA), a machine learning-based analytical pipeline that combines cosine-similarity measurement by latent cellular states with a graph-based clustering algorithm. LCA provides heuristic solutions for population number inference, dimension reduction, feature selection, and control of technical variations without explicit gene filtering. We show that LCA is robust, accurate, and powerful by comparison with multiple state-of-the-art computational methods when applied to large-scale real and simulated scRNA-seq data. Importantly, the ability of LCA to learn from representative subsets of the data provides scalability, thereby addressing a significant challenge posed by growing sample sizes in scRNA-seq data analysis.
